setup/jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
@marom-sprkl This is by default since jest 27 (you can leave it as it is I don't care)